desk/app/metagrab.hoon

```diff
@@ -105,17 +105,14 @@
   ==
 ::
 ++  fetch
-  |=  [met=?(%head %get) url=@t uas=@t]
+  |=  [met=?(%head %get) url=@t hes=header-list:http]
   ^-  card
-  =/  =header-list:http
-    :~  ['user-agent' uas]
-        ['accept' '*/*']
-    ==
+  =.  hes  [['accept' '*/*'] hes]
   =/  =request:http
-    [?-(met %head %'HEAD', %get %'GET') url header-list ~]
+    [?-(met %head %'HEAD', %get %'GET') url hes ~]
   ::NOTE  outbound-config is actually meaningless,
   ::      iris doesn't do anything with it at present...
-  [%pass /fetch/(scot %t url)/(scot %t uas)/[met] %arvo %i %request request *outbound-config:iris]
+  [%pass /fetch/(scot %t url)/(crip ~(rend co %blob hes))/[met] %arvo %i %request request *outbound-config:iris]
```

Comment:
Yes, this works, and is quite neat in that it seamlessly handles the old wire style.
I do wonder, is the only motivation here avoiding having to track this explicitly in the state, and are there other benefits or drawbacks as well?

We might never, but if we wanted to track something else pertaining to the request, it seems like we would need to forever keep a bunch of `+soft` calls at the reception and see which type fits. (Or introduce a versioned wire, ha!)